static/js/util/currency.js

```diff
@@ -1,17 +1,50 @@
 // @flow
 import cc from 'currency-codes';
 import R from 'ramda';
+import iso3166 from 'iso-3166-2';
+
+export const excludedCurrencyCodes = [
```

Comment:
yea that's my thinking too, and probably the API will only ever expand to include more currencies, so I doubt we'd run into a situation where we're allowing the user to selected a currency that's invalid on the backend.